libbeat/dashboards/dashboards.go

```diff
@@ -28,23 +28,11 @@ import (
 	"github.com/elastic/beats/libbeat/common"
 )
 
-type importMethod uint8
-
-// check import route
-const (
-	importNone importMethod = iota
-	importViaKibana
-	importViaES
-)
-
 // ImportDashboards tries to import the kibana dashboards.
-// If the Elastic Stack is at version 6.0+, the dashboards should be installed
-// via the kibana dashboard loader plugin. For older versions of the Elastic Stack
-// we write the dashboards directly into the .kibana index.
 func ImportDashboards(
 	ctx context.Context,
 	beatName, hostname, homePath string,
-	kibanaConfig, esConfig, dashboardsConfig *common.Config,
+	kibanaConfig, dashboardsConfig *common.Config,
 	msgOutputter MessageOutputter,
 ) error {
```

Comment:
Does it need an entry in the developers changelog?

Reply:
yes, will add one in my follow up PR.

Reply:
added here https://github.com/elastic/beats/pull/10450/files#diff-1fe0dd8effe994b42e3556845a93a32eR23